src/types.ts

```diff
@@ -32,7 +32,20 @@ export type LocaleInfo = {
   paths?: string[]
   hashes?: string[]
   types?: LocaleType[]
-} & LocaleObject
+} & Omit<LocaleOption, 'files'> & { files: LocaleFile[] }
+
+export type LocaleFile = { path: string; cache?: boolean }
+
+export type LocaleOption = {
```

Comment:
Can we extend with [vue-i18n-routing.d.ts](https://github.com/nuxt-modules/i18n/blob/next/src/vue-i18n-routing.d.ts)?

Reply:
I have changed the types to use the `vue-i18n-routing.d.ts` file, it seems to work for the most part. Typescript gets confused about the `file` property for some reason, so I have to add type assertions.  As seen in the duplicate utility function:
* https://github.com/nuxt-modules/i18n/pull/2357/files#diff-18f829432b971231f742904deac44b9c2b2eb287e72dc2c84684c6050305d700R522
* https://github.com/nuxt-modules/i18n/pull/2357/files#diff-39b2554fd18da165b59a6351b1aafff3714e2a80c1435f2de9706355b4d32351R405